### PR TITLE
Modify CGMES naming strategy so that the IDs are stable

### PR DESCRIPTION
**Pull request by colinepiloquet**
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

Work in progress related to #2783, also remove the generation of the id mapping csv and reading the ids from a csv.

**What kind of change does this PR introduce?**

**What is the current behavior?**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [ ] No

**If yes, please check if the following requirements are fulfilled**

- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration steps are described in the following section

**What changes might users need to make in their application due to this PR? (migration steps)**

**Other information**:

### CODE REVIEW

File: cgmes/cgmes-completion/src/main/java/com/powsybl/cgmes/completion/CreateMissingContainersPreProcessor.java [hunk 1]
```diff
@@ -71,7 +74,7 @@ public class CreateMissingContainersPreProcessor implements CgmesImportPreProces
 
     public static final String NAME = "createMissingContainers";
     public static final String FIXES_FOLDER_NAME = "iidm.import.cgmes.fixes-for-missing-containers-folder";
-    public static final double DEFAULT_NOMINAL_VALUE_FOR_MISSING_VOLTAGE_LEVELS = 100.0;
+    public static final double DEFAULT_NOMINAL_VALUE_FOR_MISSING_VOLTAGE_LEVELS = 1.2345;
```
Comment (colinepiloquet) [inline cgmes/cgmes-completion/src/main/java/com/powsybl/cgmes/completion/CreateMissingContainersPreProcessor.java:77]: What is the reason for this change?

Comment (zamarrenolm) [reply cgmes/cgmes-completion/src/main/java/com/powsybl/cgmes/completion/CreateMissingContainersPreProcessor.java:77]: We do not want to use a "valid" (reasonable) nominal voltage that could be defined

**Review by annetill — APPROVED** [review]
Thanks a lot for this huge work, it works perfectly on our french network.
